CHANGELOG.md

```diff
@@ -5,6 +5,130 @@ All notable changes to this project will be documented in this file.
+- [#5267](https://github.com/oscar-system/Oscar.jl/pull/5267) Add `order_bound` keyword argument to `subgroup_classes`
+- [#5270](https://github.com/oscar-system/Oscar.jl/pull/5270) Add an iterator of subspaces of finite vector spaces
+- [#5288](https://github.com/oscar-system/Oscar.jl/pull/5288) Add `p_rump` for `GAPGroup` and extend `torsion_subgroup` for `GAPGroup` and `WeylGroup`
+- [#5453](https://github.com/oscar-system/Oscar.jl/pull/5453) Add `character_degrees` for a given group or character table, or for given invariants of a finite abelian group and finite field
+- [#5456](https://github.com/oscar-system/Oscar.jl/pull/5456) Add `encode` and `pc_group` for encoding finite pc groups into integers and back
+- [#5532](https://github.com/oscar-system/Oscar.jl/pull/5532) Add a constructive membership test for double cosets
+
+#### Fixed bugs that resulted in unexpected errors
+
+- [#5594](https://github.com/oscar-system/Oscar.jl/pull/5594) Fix raising matrix group elements to negative powers
+
+### Lie Theory
+
+#### Only changes experimental parts of OSCAR
+
+- [#5290](https://github.com/oscar-system/Oscar.jl/pull/5290) Add `basis_coordinate_ring_kodaira_demazure`
+
+### Number Theory
+
+#### New features or extended functionality
+
+- [#5251](https://github.com/oscar-system/Oscar.jl/pull/5251) Allow `discriminant_annihilator` as input for `enumerate_classes_of_lattices_with_isometry`
+- [#5339](https://github.com/oscar-system/Oscar.jl/pull/5339) Methods for group actions on `ZZLat`
+- [#5457](https://github.com/oscar-system/Oscar.jl/pull/5457) Allow to specify several characteristic polynomials in lattice with isometry enumeration
+
+#### Performance improvements or improved testing
+
+- [#5545](https://github.com/oscar-system/Oscar.jl/pull/5545) Implement the decomposition approach for orthogonal group computation and isometry testing of integer lattices. This results in a vast speedup for lattices which are not generated by their shortest vectors.
+
+### Polyhedral Geometry
+
+#### New features or extended functionality
+
+- [#4444](https://github.com/oscar-system/Oscar.jl/pull/4444) Add transformations for polyhedral fans and complexes
+- [#4709](https://github.com/oscar-system/Oscar.jl/pull/4709) Introduce various variants of `halfspace_matrix_pair(ZZ, ...)` for polyhedral objects to access facets etc as matrices
+- [#5372](https://github.com/oscar-system/Oscar.jl/pull/5372) Add `castelnuovo_excess` and `is_castelnuovo` for lattice polytopes
+- [#5384](https://github.com/oscar-system/Oscar.jl/pull/5384) polyhedral geometry: doc enhancements and further functions inherited from polymake 
+
+### Toric Geometry
+
+#### New features or extended functionality
+
+- [#5032](https://github.com/oscar-system/Oscar.jl/pull/5032) Improve vector of minimal exponents method for cohomology computations
+- [#5512](https://github.com/oscar-system/Oscar.jl/pull/5512) **Breaking:** Rename `all_cohomologies` to `sheaf_cohomology`
```

Comment:
This is actually not breaking as it introduces deprecations. I'll update the wording on the PR so that we get it here after the next iteration